content/en/blog/_posts/2025-XX-XX-endpoints-deprecation.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Endpoints formally deprecated in favor of EndpointSlices"
+slug: endpoints-deprecation
+date: 2025-XX-XX
+draft: true
+author: >
+  Dan Winship (Red Hat)
+---
+
+Since the addition of [EndpointSlices] several years ago, the
+Endpoints API in Kubernetes has been gathering dust. New Service
+features like [dual-stack networking] and [topology-aware routing] are
+only supported via the EndpointSlice API, so all service proxies,
+Gateway API implementations, and similar controllers have had to be
+ported from using Endpoints to using EndpointSlices. At this point,
+the Endpoints API is really only there to avoid breaking end user
+workloads and scripts that still make use of it.
+
+We are now beginning the process of officially deprecating the
+Endpoints API, with clarifications to the documentation, and warnings
+from the API server when a client reads or writes Endpoints resources.
```

Comment:
```suggestion
We are deprecating the Endpoints API. This will initially involve documentation updates and API server warnings when Endpoints resources are accessed.
```

Wording simplified slightly

Reply:
Also see https://github.com/kubernetes/website/pull/49976/files#r2007454583

Reply:
That rewording seems a little abrupt. I tried rearranging and rewording some things...